.npmignore

```diff
@@ -0,0 +1,9 @@
+jobs/
+node_modules/
+.gitignore
+.npmrc
+CODEOWNERS
+SECURITY.md
+*.ts
+*.json
```

Comment:
Isn't package.json needed? Won't `*.json` match that?

Reply:
Should only `tsconfig.json` be ignored?

Reply:
Yes, I believe you are right.